packages/api/package.json

```diff
@@ -5,6 +5,7 @@
   "author": "David Adams",
   "license": "AGPL-3.0-or-later",
   "devDependencies": {
+    "@types/cors": "^2.8.12",
```

Comment:
why was this needed to be added here?

Reply:
Good question. The api build began failing because it couldn't resolve the types for the cors package. My (and @daywiss's) assumption is that this was previously getting this dep via some nested package that was installed under truffle, so once it was removed this began failing.

Anyway, the package uses the cors types, so it should be including them as a dep. It was a little odd to see it sporadically fail like this though.